backend/src/main/java/wooteco/prolog/studylog/application/StudylogService.java

```diff
@@ -64,6 +68,49 @@ public StudylogsResponse findStudylogs(StudylogsSearchRequest request, Long memb
         return studylogs;
     }
 
+    public StudylogsResponse findMostPopularStudylogs(
+        Pageable pageable,
+        Long memberId,
+        boolean isAnonymousMember
+    ) {
+        List<Studylog> studylogs = findStudyLogsByDays(pageable, LocalDateTime.now());
```

Comment:
```suggestion
        List<Studylog> studylogs = findStudylogsByDays(pageable, LocalDateTime.now());
```

스터디로그(Studylog) 대소문자 주의해주시고 작업하신 부분 내에서도 함께 고쳐주세요~

Reply:
작업 부분과 프로젝트 전반적으로 `StudyLog`로 사용중인 부분 모두 `Studylog`로 변경해보았습니당!